p256-crypto/src/core.rs

```diff
@@ -83,23 +83,7 @@ pub mod p256 {
         }
     }
 
-    impl TraitPublic for Public {
-        fn to_public_crypto_pair(&self) -> CryptoTypePublicPair {
-            CryptoTypePublicPair(CRYPTO_ID, self.to_raw_vec())
-        }
-    }
-
-    impl From<Public> for CryptoTypePublicPair {
-        fn from(key: Public) -> Self {
-            (&key).into()
-        }
-    }
-
-    impl From<&Public> for CryptoTypePublicPair {
-        fn from(key: &Public) -> Self {
-            CryptoTypePublicPair(CRYPTO_ID, key.to_raw_vec())
-        }
-    }
```

Comment:
I am not sure about this changes. The `CRYPTO_ID` is important since we are adding support for a new curve here.

Reply:
CryptoTypePublicPair was removed